autogen/agentchat/contrib/agent_eval/README.md

```diff
@@ -0,0 +1,7 @@
+Agents for running the AgentEval pipeline.
+
+AgentEval is a process for evaluating a LLM-based system's performance on a given task.
+
+When given a task to evaluate and a few example runs, the critic and subcritic agents create evaluation criteria for evaluating a system's solution. Once the criteria has been created, the quantifier agent can evaluate subsequent task solutions based on the generated criteria.
+
+For more information see: [AgentEval Integration Roadmap](https://github.com/microsoft/autogen/issues/2162)
```

Comment:
i dont think we will need a separate readme file for the agent_eval.

Reply:
I'm going to leave for now as it adds discoverability to AgentEval. If we find it becomes unneccessary, will remove.